### ISSUE TITLE
Users are having issues with CUE 2

### ISSUE THREAD

**Issue by antonpup** [antonpup/Aurora#148]
Users are reporting that using CUE 2 does not automatically detect the Corsair SDK.

**Comment by brainbug89** [comment]
FYI I have no problems with CUE 2.3.74 and 2.4.66

(Win 7, x64)

**Comment by antonpup** [comment]
I have pushed a small update that fixes issues with CUE 2.
[Version 0.5.1b](https://github.com/antonpup/Aurora/releases/tag/v0.5.1b)

**Comment by IMNdi** [comment]
I don't know if this is the place to do it, but I have updated from 0.5.1a to b and it stopped working. I reverted to .a and it is working again.

.b has the "Could not start" try as administrator thing; I believe it was the thing it was supposed to fix. The CUE thing reports it is up to date (1.16.42).

I have a G910 KBD, Corsair mouse (M65 RGB) and headphones (Void RGB).

It all works just fine in 0.5.1a, including colors, though there are some minor issues. 

[2016_15_09.txt](https://github.com/antonpup/Aurora/files/504221/2016_15_09.txt)

Windows 10 Pro (10.0.14393)
CUE (1.16.42)
Aurora (0.5.1b)

**Comment by brainbug89** [comment]
Maybe try the lastest CUE version 2.5.66

**Comment by IMNdi** [comment]
Oddly enough, the updater claims there is no newer version. I wonder if it's an error on Corsair's side or it it has some issues to resolve.

I saw the new version on their website. I will try it tomorrow.

**Comment by antonpup** [comment]
My CUE also fails to retrieve an update. My version's 2.4.66. I haven't had many issues with the dev version, the only problem I saw was that CUE has problems initializing at windows launch. But that's about it, pressing "Retry for connected devices" fixes it.

**Comment by brainbug89** [comment]
CUE update is disabled for open beta of CUE2

**Comment by IMNdi** [comment]
Checking in.

CUE 2.5.66
AURORA 0.5.1b

Aurora starts and works as expected but will not detect Corsair hardware. Software (CUE) is installed and working (I have reconfigured my old colors) and SDK is ON in settings, but Aurora says no connected devices. Keeps retrying and ultimately fails.

I also tried a manual retry about 5 minutes in, just to make sure. No connected devices for Corsair.

G910 is still humming along nicely (minus the known G keys issue).
